http.filesystem.config.server/chart/templates/service-zilla-config.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: zilla-config
+  labels:
+    app.kubernetes.io/instance: zilla-config
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: zilla-config
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/instance: zilla-config
+    spec:
+      containers:
+        - name: zilla-config
+          # TODO: change to latest when it gets merged
+          image: "ghcr.io/aklivity/zilla:develop-SNAPSHOT"
+          args: ["start", "-v", "-e"]
+          env:
+            - name: KEYSTORE_PASSWORD
+              value: generated
+            - name: JAVA_TOOL_OPTIONS
+              value: '-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005'
+          ports:
+            - containerPort: 8081
+            - containerPort: 5005
+            - containerPort: 9091
+          readinessProbe:
+            tcpSocket:
+              port: 8081
+            initialDelaySeconds: 3
+            periodSeconds: 5
+          volumeMounts:
+            - name: config
+              mountPath: /etc/zilla
+            - name: served-files
+              mountPath: /var/www
+            - name: tls
+              mountPath: /etc/zilla/tls
+      volumes:
+        - name: config
+          configMap:
+            name: zilla
+        - name: served-files
+          projected:
+            sources:
+            - configMap:
+                name: configmap-served
+            - secret:
+                name: tls-served
+                items:
+                  - key: localhost.p12
+                    path: localhost.p12
+        - name: tls
+          secret:
+            secretName: tls
+            items:
+              - key: localhost.p12
+                path: localhost.p12
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: zilla
+data:
+  zilla.yaml: |-
+{{ .Files.Get "files/zilla.yaml" | indent 4 }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: configmap-served
```

Comment:
Suggest renaming to `zilla-served` for parity with `zilla`, much like `tls` vs `tls-served` pattern below.